openedx/core/djangoapps/bookmarks/models.py

```diff
@@ -60,6 +61,29 @@ def __unicode__(self):
         return self.resource_id
 
     @classmethod
+    def can_create_more(cls, data):
+        """
+        Determine if a new Bookmark can be created for the course
+        based on limit defined in django.conf.settings.MAX_BOOKMARKS_PER_COURSE
+
+        Arguments:
+            data (dict): The data to create the object with.
+        Returns:
+            Boolean
+        """
+        max_bookmarks_per_course = settings.MAX_BOOKMARKS_PER_COURSE
+        data = dict(data)
+
+        user = data['user']
+        course_key = data['usage_key'].course_key
+
+        # User can create up to max_bookmarks_per_course bookmarks
+        if cls.objects.filter(user=user, course_key=course_key).count() == max_bookmarks_per_course:
```

Comment:
This will ignore bookmarks with > 100 values, Change the conditional to `cls.objects.filter(user=user, course_key=course_key).count() >= max_bookmarks_per_course`